en/developer_guide/navigation_panel.rst

```diff
@@ -43,7 +43,11 @@ The menu link must have the following attributes :
 
 It can also have data-* attributes that can help you to define custom properties to be used in a Backbone route,
 for example :
+
 * data-url : API url to be called to retrieve the data to be displayed
+* data-translated-header : the list of column names in dataTable view (see `Entity list`_)
+* data-visible-elements : the list of columns, visible by default, in dataTable view (see `Entity list`_)
+* data-displayed-elements : the list of all columns in dataTable view (see `Entity list`_)
```

Comment:
I also think the wording should be more specific, as each definition is more or less the same here. Some ideas :
data-translated-header : the list of column labels ?
data-visible-elements : the list of columns identified by id ?
data-displayed-elements : list of all columns identified by id ?